### PR TITLE
Allen Coral Atlas (ACA) v2 ingest

### PR DESCRIPTION
**Pull request by mitchest**
New PR because original couldn't seem to get past the initial CLA check fail

Changes include:
- new .jsonnet file for v2 data
- updates paths for predecessor/successor/latest 
- updated links for v1 + v2 data throughout

### CODE REVIEW

File: catalog/ACA/ACA_reef_habitat_v2_0.jsonnet [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+    },
+    {
+      rel: ee_const.rel.cite_as,
+      href: 'https://doi.org/10.5281/zenodo.3833242',
+    },
+  ],
+  keywords: [
+    'coral',
+    'ocean',
+    'planet',
```
[the remark this answers is not in the document]
Comment (mitchest) [reply catalog/ACA/ACA_reef_habitat_v2_0.jsonnet]: ok done for both files

**Review by simonff — APPROVED** [review]

File: catalog/ACA/ACA_reef_habitat_v1_0.jsonnet [hunk 2]
```diff
@@ -73,7 +75,7 @@ local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
   keywords: [
     'coral',
     'ocean',
-    'planet',
+    'sentinel2_derived',
```
Comment (schwehr) [inline catalog/ACA/ACA_reef_habitat_v1_0.jsonnet:78]: Please sort the keywords

Issue('ACA/reef_habitat/v1_0', 'ACA/ACA_reef_habitat_v1_0.json', 'keywords', '"keywords" must be sorted', IssueLevel.error)

File: catalog/ACA/ACA_reef_habitat_v2_0.jsonnet [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+    {
+      rel: ee_const.rel.source,
+      href: 'https://storage.googleapis.com/coral-atlas-user-downloads-prod/Global-Dataset-20211006223100.zip',
+    },
+    {
+      rel: ee_const.rel.cite_as,
+      href: 'https://doi.org/10.5281/zenodo.3833242',
+    },
+  ],
+  keywords: [
```
Comment (schwehr) [inline catalog/ACA/ACA_reef_habitat_v2_0.jsonnet:85]: Please sort the keywords

Issue('ACA/reef_habitat/v2_0', 'ACA/ACA_reef_habitat_v2_0.json', 'keywords', '"keywords" must be sorted', IssueLevel.error)

File: catalog/ACA/catalog.jsonnet [hunk 4]
```diff
@@ -19,5 +19,6 @@ local self_url = base_url + base_filename;
     ee.link.parent(parent_url),
     ee.link.self_link(self_url),
     ee.link.child_collection('ACA_reef_habitat_v1_0', base_url),
+    ee.link.child_collection('ACA_reef_habitat_v2_0', base_url),
```
Comment (schwehr) [inline catalog/ACA/catalog.jsonnet:22]: Please comment out this line and add a TODO as follows:

```javascript
// TODO(schwehr): Enable when the dataset is ready to be live.
// ee.link.child_collection('ACA_reef_habitat_v2_0', base_url),
```

Issue('ACA/reef_habitat/v2_0', 'ACA/ACA_reef_habitat_v2_0.json', 'parent_child', 'Child link when gee:skip_indexing is true: ACA/catalog

File: catalog/ACA/ACA_reef_habitat_v2_0.jsonnet [hunk 5]
```diff
@@ -0,0 +1,322 @@
+local id = 'ACA/reef_habitat/v2_0';
```
Comment (schwehr) [inline catalog/ACA/ACA_reef_habitat_v2_0.jsonnet:1]: Please remove trailing space characters.  Some text editors can do this.  Or you can some somethings like:

```shell
perl -pi -e 's/\s+\n/\n/g' catalog/ACA/ACA_reef_habitat_v2_0.jsonnet
```

**Review by schwehr — CHANGES_REQUESTED** [review]

**Review by simonff — APPROVED** [review]

File: catalog/ACA/ACA_reef_habitat_v2_0.jsonnet (hunk 5 above)
Comment (mitchest) [reply catalog/ACA/ACA_reef_habitat_v2_0.jsonnet:1]: OK, done as best i can tell

**Review by schwehr — APPROVED** [review]

**Pull request merged**